stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -64,17 +67,28 @@ public fun MessageText(
     val styledText = buildAnnotatedMessageText(message)
     val annotations = styledText.getStringAnnotations(0, styledText.lastIndex)
 
+    val isMyMessage = message.isMine()
+
+    val isEmojiOnly = message.isEmojiOnly()
+    val isSingleEmoji = message.isSingleEmoji()
+
+    val style = when {
```

Comment:
I suggest adding this on top of the when:

```
// TODO: Fix emoji font padding once this is resolved and exposed: https://issuetracker.google.com/issues/171394808
```

Because currently we have some ugly font padding around the emoji as it's a large font size and that makes the reactions header a bit too far off from the emoji. Granted, we could rework the reactions header to look nicer and point to the message below but that's another issue.

Reply:
![Screenshot_1650519072](https://user-images.githubusercontent.com/17215808/164381141-c8c03cd8-5fd2-4122-a0e8-0680ea2e88f8.png)